website/content/docs/extending/plugins.mdx

```diff
@@ -257,6 +257,39 @@ with "packer-builder-custom". Log output is _extremely_ helpful in debugging
 issues and you're encouraged to be as verbose as you need to be in order for
 the logs to be helpful.
 
+### Creating a GitHub Release
+
+To create a GitHub release with the assets that will be consumed by Packer, we provide a pre-defined release workflow configuration
+using [GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions). We strongly encourage maintainers to use this configuration
+to make sure the release contains the assets expected by Packer to load and use a plugin.
+[GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions) allow you to execute workflows when events on your repository occur.
+You can use this to create releases whenever a new version tag is created on your repository.
+
+Here's what you need to create releases using GitHub Actions:
+1. Generate a GPG key to be used when signing releases (See [GitHub's detailed instructions](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/generating-a-new-gpg-key)
+for help with this step)
+2. Copy the [GoReleaser configuration from the packer-plugin-scaffolding repository](https://github.com/hashicorp/packer-plugin-scaffolding) to the root of your repository.
+3. Copy the [GitHub Actions workflow from the packer-plugin-scaffolding repository](https://github.com/hashicorp/packer-plugin-scaffolding) to `.github/workflows/release.yml` in your repository.
+4. Go to Settings > Secrets in your repository, and add the following secrets:
+    - `GPG_PRIVATE_KEY` - Your ASCII-armored GPG private key. You can export this with `gpg --armor --export-secret-keys [key ID or email]`.
+    - `PASSPHRASE` - The passphrase for your GPG private key.
+5. Push a new valid version tag (e.g. `v1.2.3`) to test that the GitHub Actions releaser is working. The tag must be a valid [Semantic Version](https://semver.org/) preceded with a `v`.
+6. Make sure to [register your plugin](#registering-the-plugin) with us.
```

Comment:
All plugins must be registered then ?

Reply:
We would like to, so we can publish then to releases.hashicorp.com

Reply:
In the Registering the Plugin section I made it clear that the purpose of that is to publish the release to releases.hashicorp.com. The plugins will work without registering them but I don't want to encourage people not to do it.